src/seedu/addressbook/AddressBook.java

```diff
@@ -484,14 +481,37 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
      */
     private static ArrayList<String[]> getPersonsWithNameContainingAnyKeyword(Collection<String> keywords) {
         final ArrayList<String[]> matchedPersons = new ArrayList<>();
+        keywords = toLowerCase(keywords);
+        
         for (String[] person : getAllPersonsInAddressBook()) {
-            final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person)));
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person).toLowerCase()));
+            final boolean hasCommonWord = !Collections.disjoint(wordsInName, keywords);
+            if (hasCommonWord) {
                 matchedPersons.add(person);
             }
         }
         return matchedPersons;
     }
+    
+    /**
+     * Make all alphabets in given strings lowercase
+     * 
+     * @param stringd to be converted to lowercase
+     * @return converted lowercase strings
+     */
+    private static ArrayList<String> toLowerCase(Collection<String> strings) {
+    	ArrayList<String> lowercaseStrings = new ArrayList<String>();
+    	Iterator<String> strIter = strings.iterator();
+    	
+    	while(strIter.hasNext()) {
+    		String tempString = strIter.next();
+    		lowercaseStrings.add(tempString.toLowerCase());
+    	}
+    	
+    	return lowercaseStrings;
+    }
+    
```

Comment:
Next would be to do up the necessary documentations and tests for the modified behavior.